product_docs/docs/pgd/5/reference/functions-internal.mdx

```diff
@@ -156,19 +156,27 @@ bdr.difference_fix_xact_set_avoid_conflict()
 
 ### `bdr.drop_node`
 
-Drops a node.
-
-!!! Warning
-    This function isn't intended for regular use. Execute it only
-    if instructed by Technical Support.
+Drops a node's metadata.
 
 This function removes the metadata for a given node from the local
 database. The node can be either:
 
--   The local node, in which case all the node metadata is removed,
-    including information about remote nodes.
--   A remote node, in which case only metadata for that specific
-    node is removed.
+-   The local node, in which case it removes all the node metadata, including information about remote nodes.
+-   A remote node, in which case it only removes metadata for that specific node.
+
+
+
+!!! Important When to use `bdr.drop_node()`
+Because PGD can reuse existing node names providing the node name belongs to a node in a `PARTED` state, you should not use `bdr.drop_node` to remove the original node metadata.
```

Comment:
How about this?
```suggestion
Because PGD can reuse existing node names, you should not use `bdr.drop_node` to remove the original node metadata, unless the node name belongs to a node in a `PARTED` state.
```

Reply:
No, that's the wrong way around. 

```suggestion
PGD can reuse existing node names providing the node name belongs to a node in a `PARTED` state. To reuse a node name, use `bdr.part_node()` to remove the original node from the cluster. You should not use `bdr.drop_node` to remove the original node metadata. 
```